src/pages/Detail/Picker.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { ko } from 'date-fns/esm/locale';
+
+const Picker = props => {
+  const { setTotalPrice, totalPrice, price } = props;
```

Comment:
매개변수자리에서 바로 구조 분해 할당 할 수 있습니다!
```suggestion
const Picker = ({ setTotalPrice, totalPrice, price }) => {
```